### PR TITLE
Using default number of procs in Unix

### PR DESCRIPTION
**Pull request by roigcarlo**
Funny enough, my OS decided to replace it while writing the commit message :smile:

### CODE REVIEW

**Review by loumalouomega — DISMISSED** [review]
OK

File: scripts/standard_configure.sh [hunk 1]
```diff
@@ -45,4 +45,4 @@ rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/CMakeFiles"
 cmake -H"${KRATOS_SOURCE}" -B"${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" -DUSE_MPI=OFF
 
 # Buid
-cmake --build "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" --target install -- -j4
+cmake --build "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" --target install -- -j$(nproc)
```
Comment (philbucher) [inline scripts/standard_configure.sh:48]: `nproc` is an env-var in unix?

Comment (philbucher) [reply scripts/standard_configure.sh:48]: not defined on my machine :eyes:

Comment (roigcarlo) [reply scripts/standard_configure.sh:48]: It is a command, and should be unix standard. the problem is I don't think everyone will get what they expect. For what I've been able to read, this returns the number of procs available to the process calling it. For example, my computer limits the number of cpu that a terminal can use to one, so whenever I use it it always fails to 1 (Even when I have 4).

I suppose that's the reason I had id hardcoded to 4 when I uploaded it in the first place.

**Comment by roigcarlo** [comment]
@RiccardoRossi @maceligueta @AlejandroCornejo I added the set install links for win here as is related with this PR.

File: scripts/standard_configure.bat [hunk 2]
```diff
@@ -16,6 +16,7 @@ rem Set variables
 set KRATOS_SOURCE=~0,-1%/..
 set KRATOS_BUILD=%KRATOS_SOURCE%/build
 set KRATOS_APP_DIR=applications
+rem set KRATOS_INSTALL_PYTHON_USING_LINKS=ON
```
Comment (philbucher) [inline scripts/standard_configure.bat:21]: I thought thats not working on Win?

Comment (roigcarlo) [reply scripts/standard_configure.bat:21]: Apparently it works if you use an elevated prompt

Comment (maceligueta) [reply scripts/standard_configure.bat:21]: @AlejandroCornejo  managed to do it 
https://github.com/KratosMultiphysics/Kratos/issues/6002#issuecomment-566604861
But only with admin permissions
https://github.com/KratosMultiphysics/Kratos/issues/6002#issuecomment-566624369

Comment (philbucher) [reply scripts/standard_configure.bat:21]: Then I think it makes sense to add a comment telling this

Comment (roigcarlo) [reply scripts/standard_configure.bat:21]: Done

**Comment by pooyan-dadvand** [comment]
Fine for me! :+1:

**Comment by roigcarlo** [comment]
@philbucher @pooyan-dadvand Should we merge this? (If so someone can approve?)

**Review by philbucher — APPROVED** [review]
Ok

**Pull request merged**